main.tf

```diff
@@ -1,3 +1,12 @@
-/********************************************************************
-This file is used to implement the ROOT module.
-*********************************************************************/
+##############################################################################
+# SCC module
+##############################################################################
+
+resource "ibm_resource_instance" "scc_instance" {
+  name              = var.instance_name
+  service           = "compliance"
+  plan              = var.plan
+  location          = var.region
+  resource_group_id = var.resource_group_id
+  tags              = var.resource_tags
+}
```

Comment:
I think at least the initial version of this module should include the ability to hook the SCC instance up with an existing COS bucket. Can you check if its possible to do that in terraform? Without the bucket, the instance can't function. 

Also bare in mind that an s2s auth policy is required to allow the instance to write to COS, so the module should support creating that too.

Reply:
as discussed, will follow up COS bucket support in a separate PR